egs/aspire/s5/local/nnet3/run_ivector_common.sh

```diff
@@ -37,16 +52,21 @@ if [ $stage -le 1 ]; then
     else
       num_reps=1
     fi
-    reverb_data_dirs=
-    for i in `seq 1 $num_reps`; do
-      cur_dest_dir=" data/temp_${data_dir}_${i}"
-      local/multi_condition/reverberate_data_dir.sh --random-seed $i \
-        --snrs "$snrs" --log-dir exp/make_corrupted_wav \
-        data/${data_dir}  data/impulses_noises $cur_dest_dir
-      reverb_data_dirs+=" $cur_dest_dir"
-    done
-    utils/combine_data.sh --extra-files utt2uniq data/${data_dir}_rvb $reverb_data_dirs
-    rm -rf $reverb_data_dirs
+
+    python steps/data/reverberate_data_dir.py \
+      --prefix "rev" \
+      --rir-set-parameters "1.0, SIM_RIR/simulated_rir_mediumroom/rir_list" \
```

Comment:
I would recommend adding your swbd recipes where you have examples of combining several rir sets and noise sets. 

Also please do not modify the original reverberation recipe as we are maintaining it for comparison with our ASpIRE submission. These experiments are very difficult to repeat and I would prefer not modifying this recipe until you have results which compare real rir+isotopic noise, real rir + sim rir + isotropic noise + point source noise settings.

If you would like to commit this new data prep recipe right now, you could move data preparation to another script and add the option of just using simulated rirs in that script.

Also remember to add the results with new recipe.